templates/packageVersion.mustache

```diff
@@ -66,6 +60,31 @@
           <p>None</p>
           {{/cabalFileRevisions}}
         </dd>
+        {{#pkgDesc.library}}
+        <dt class="col-sm-3">Dependencies</dt>
+        <dd class="col-sm-9">{{#build-depends}}{{#.}}{{.}}<br>{{/.}}{{/build-depends}}</dd>
```

Comment:
```suggestion
        <dd class="col-sm-9"><ul>{{#build-depends}}{{#.}}<li>{{.}}</li>{{/.}}{{/build-depends}}</ul></dd>
```
seems more semantic? And then you don't have the problem of having an extra trailing `<br>` since you just delimit each element.

Applies throughout.

Reply:
Can possibly get a nice display like on Hackage with something like this: https://stackoverflow.com/questions/1517220/how-to-style-unordered-lists-in-css-as-comma-separated-text